system-tests/pom.xml

```diff
@@ -209,7 +224,45 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.6.1</version>
+                <executions>
+                    <execution>
+                    <!-- Needs to run before java-generator-maven-plugin -->
+                    <phase>generate-sources</phase>
+                    <goals>
+                        <goal>wget</goal>
+                    </goals>
+                    <configuration>
+                        <url>https://raw.githubusercontent.com/Apicurio/apicurio-registry-operator/1.0.0-v2.0.0.final/packagemanifests/1.0.0-v2.0.0.final/registry.apicur.io_apicurioregistries.yaml</url>
```

Comment:
@jsenko this is the latest "release" I found, do you think that the tag is accurate?